lib/msal-browser/package.json

```diff
@@ -10,7 +10,7 @@
         "type": "git",
         "url": "https://github.com/AzureAD/microsoft-authentication-library-for-js.git"
     },
-    "version": "0.0.1",
+    "version": "2.0.0-alpha",
```

Comment:
`2.0.0-alpha.0` to be consistent with how we have been doing pre-release versions.

Reply:
done.